community-operators/postgresql/postgres-operator.v4.0.1.clusterserviceversion.yaml

```diff
@@ -77,57 +178,70 @@ spec:
     You can generate new self-signed certificates using scripts in the operator repository.
 
     ```
-    export COROOT=$(pwd)
-    cd $COROOT/deploy
-    $COROOT/deploy/gen-api-keys.sh
-    $COROOT/deploy/gen-sshd-keys.sh
-    cd $COROOT
+    export PGOROOT=$(pwd)
+    cd $PGOROOT/deploy
+    $PGOROOT/deploy/gen-api-keys.sh
+    $PGOROOT/deploy/gen-sshd-keys.sh
+    cd $PGOROOT
     ```
 
     ### Configuration and Secrets
 
     Once the configuration changes have been updated and certificates are in place, we can save the information to the cluster.
 
+    Create the pgo namespace if it does not exist already. This single namespace is where the operator should be deployed to.
+    
+    ```
+    oc create namespace pgo 
+    ```
+
+    Create the pgouser1 namespace if it does not exist. This is the namespace the operator will watch based on the default configuration.
+
+    ```
+    oc create namespace pgouser1
```

Comment:
Use of two namespaces, pgo and pgouser1 seems unusual. Do you plan to support installMode MultiNamespace in the future and this is an incremental step towards that end? Until the MultiNamespace installMode is supported, it would be seem much simpler to support a single namespace into which the operator is deployed and which that same operator watches.

Reply:
Postgres Operator is installed in the 'pgo' namespace by convention and it then watches one or more other namespaces where it deploys clusters. We will revisit the multi-namespace installation here in a future revision.

Reply:
With the current configuration, how do you make the operator watch more than one namespace per cluster? Is it possible to install just the Postgres Operator pre-requisites into the 'pgo' namespace and then allow the use of a subscription to install the Operator into each namespace of interest? Then each Operator in each namespace would watch only its own namespace.

Reply:
currently, the NAMESPACE variable within the container determines what namespace(s) to watch. A comma separated list is supported. See here for details:

https://crunchydata.github.io/postgres-operator/stable/gettingstarted/design/namespace/